### ISSUE TITLE
Question about your examples

### ISSUE THREAD

**Issue by mng8** [ixaxaar/pytorch-dnc#55]
Why do you not use controller hidden or read vectors over time as your perform the task?

**Comment by ixaxaar** [comment]
The examples are from the paper. 

Personally, I did use the controller hidden vectors as well as let the memory remain uncleared for entire training time while doing various experiments myself and many other variations of these.

**Comment by mng8** [comment]
Looking at inside the model code, I do see you use the reads and controller hidden vector inside the time iterations

**Comment by ixaxaar** [comment]
Yes, one has to use them within timesteps otherwise the whole thing is just a bunch of vanilla RNNs i think.

What I was talking about is entire training time across batches and epochs, for e.g. using the memory as a global knowledge base of sorts.
